pkg/extensions/extension_mgmt.go

```diff
@@ -70,6 +70,15 @@ type mgmt struct {
 	log    log.Logger
 }
 
+// mgmtHandler godoc
+// @Summary Get current server configuration
+// @Description Get current server configuration
+// @Router 	/v2/_zot/ext/mgmt [get]
+// @Accept  json
+// @Produce json
+// @Success 200 {object} 	extensions.StrippedConfig
+// @Failure 404 {string} 	string 				"not found"
```

Comment:
I think we should remove this 404 and 400 responses, mgmt shouldn't respond with them in any scenario.

also add 500 internal server error

200 and 500 should be the only responses

Reply:
I made the changes